SD_DSC.adoc

```diff
@@ -1947,6 +1947,8 @@ Test 1: The evaluator shall verify that the DSC performs an integrity check of a
 
 Test 2: The evaluator shall ensure that when an integrity check failure occurs specific to failing KATs and failure to verify the integrity of the TSF, the TOE will prevent any further processing of the current TSF and user data. 
 
+Test 3: [CONDITIONAL] For each condition where a self test may be initiated, the evalautor shall verify that the self tests are run as specified.
+
```

Comment:
"evalautor" typo
Also, following other tests, maybe we should write `Test 3 [conditional]:`? I know there's an open issue regarding the consistency for Milestone v3.0.